manifests/o/ojdkbuild/ojdkbuild/1.8.3321.9/ojdkbuild.ojdkbuild.locale.en-US.yaml

```
@@ -1,15 +1,15 @@
-# Created using wingetcreate 1.1.2.0
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
-
-PackageIdentifier: ojdkbuild.ojdkbuild
-PackageVersion: 1.8.0.3321.9
-PackageLocale: en-US
-Publisher: ojdkbuild open-source project
-PackageName: ojdkbuild OpenJDK JRE 8
-PackageUrl: https://github.com/ojdkbuild/ojdkbuild/wiki/Motivation
-License: GPL 2 with Classpath Exception
-LicenseUrl: https://github.com/ojdkbuild/ojdkbuild/blob/master/LICENSE
-ShortDescription: Community builds using source code from OpenJDK project
-ManifestType: defaultLocale
-ManifestVersion: 1.2.0
-
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: ojdkbuild.ojdkbuild
+PackageVersion: 1.8.3321.9
+PackageLocale: en-US
+Publisher: ojdkbuild open-source project
+PackageName: ojdkbuild OpenJDK JRE 8
+PackageUrl: https://github.com/ojdkbuild/ojdkbuild/wiki/Motivation
```

```suggestion
PackageUrl: https://github.com/ojdkbuild/ojdkbuild/
```
Wiki can be mentioned in the `Documentations` section.